doc/source/ray-core/patterns/return-ray-put.rst

```diff
@@ -0,0 +1,24 @@
+Anti-pattern: Returning ray.put() ObjectRefs from a task harms performance and fault tolerance
+==============================================================================================
+
+**TLDR:** Avoid calling :ref:`ray.put() <ray-put-ref>` on task return values and returning the resulting ObjectRefs;
+returning these values directly via :ref:`static <ray-task-returns>` and :ref:`dynamic <dynamic-generators>` ``num_returns``.
+
+Returning ray.put() ObjectRefs are considered anti-patterns for the following reasons:
+
+- It disallows inlining small return values: Ray has a performance optimization to return small (<= 100KB) values inline directly to the owner, avoiding going through the distributed object store.
```

Comment:
Avoid jargon like "owner".

Reply:
I can change to caller here. But later on we will discuss fault tolerance, should we also avoid "owner"?